components/Comment/CommentAvatars.tsx

```diff
@@ -0,0 +1,34 @@
+import AuthorFacePile from "../shared/AuthorFacePile";
+import { css, StyleSheet } from "aphrodite";
+import { AuthorProfile, ID } from "~/config/types/root_types";
+
+type CommentAvatarsArgs = {
```

Comment:
Should we move this out of this file, or leave it for simplicity? 

I'm a fan of leaving things that don't need to be exported in one file so you don't need to open 2 files to do the same operation.

Reply:
You know, you are probably right. Initially this file was `CommentAuthors` which had more stuff in it